packages/table/src/headers/columnHeader.tsx

```diff
@@ -103,18 +103,13 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
     }
 
     private wrapCells = (cells: Array<React.ReactElement<any>>) => {
-        const { grid, viewportRect, columnIndexStart } = this.props;
+        const { grid } = this.props;
```

Comment:
how does this affect the computation of the viewport rect and which cells are visible?